### DISCUSSION TITLE
Add last modification user for product

### DISCUSSION THREAD

**Discussion by ufoloko** [openstore-ecommerce/OpenStore#304] [Ideas]
Actually module have last modification date for an item (product, category, etc), it should be great to have last modification user id too, this can allow to know who made last modifications, I think this is important for auditing

**Comment by leedavi** [comment]
Sorry about the last response.  The message got lost.

The Last Modified date is added by the DB update.  I agree adding the user email would be a great advantage for audit.  Be sometimes the update is automatic by the scheduler and plugins.  This would give us issues about the email, but not impossible.

However, the change would mean a change to the DB structure.  I think a change like this would need to be a new plugin for the system, events could create a full audit trail for each product.

To be honest, I don't think this kind of thing will be created unless we get some funding for it.
